src/js/views/patients/worklist/filters_views.js

```diff
@@ -1,55 +0,0 @@
-import hbs from 'handlebars-inline-precompile';
-import { View } from 'marionette';
-
-import 'scss/modules/buttons.scss';
-
-import Droplist from 'js/components/droplist';
-
-import 'js/views/patients/worklist/worklist-list.scss';
-
-const FiltersView = View.extend({
-  className: 'worklist-list__filters',
-  template: hbs`
-    <div class="worklist-list__all-filters" data-all-filters-region></div>
-    <div class="worklist-list__filter" data-workspace-filter-region></div>
-    <div class="worklist-list__filter" data-owner-toggle-region></div>
-  `,
-  regions: {
-    allFilters: '[data-all-filters-region]',
-    workspace: '[data-workspace-filter-region]',
-    ownerToggle: '[data-owner-toggle-region]',
```

Comment:
This `ownerToggle` region was moved to the `ListTitleView` a while ago, but wasn't removed from the `FiltersView` here. So I removed that region when I moved these views over to the `worklist_views.js` file.